addons/docs/src/lib/docgen/extractDocgenProps.test.ts

```diff
@@ -93,6 +93,25 @@ TypeSystems.forEach((x) => {
         expect(propDef.required).toBe(false);
         expect(propDef.defaultValue.summary).toBe('"Default"');
       });
+
+      it('should map defaults docgen info properly, vue', () => {
+        const component = createComponent({
+          ...createStringType(x),
+          description: 'Hey! Hey!',
+          defaultValue: {
+            value: "'Default'",
+            func: false,
```

Comment:
Is `func` always `false` in these cases?

Reply:
I suppose `func` is probably true when you are using one of them vue default value "generators". I haven't explored the full space of everything vue can do though :/